### DISCUSSION TITLE
console get "scrambled" when scrolling

### DISCUSSION THREAD

**Discussion by brndngrhm** [posit-dev/positron#4477] [Q&A] [answered]
This started happening after working in positron all day. At some point i scrolled up in the console and text appeared all scrambled as if the previous output wa being cumulatively layered on top as i scroll. When i scroll back down, the text appars normal again.
No Scroll: 
![image](https://github.com/user-attachments/assets/5d5855f6-72ba-4b59-b2a2-0936774514fb)

After Scrolling:
![image](https://github.com/user-attachments/assets/9da01df5-df2e-44e8-ab8a-5ab242bbadd0)

**Comment by juliasilge** [comment, marked as answer]
Thank you for this report! 🙌  We've got this tracked in https://github.com/posit-dev/positron/issues/2116 and plan to work on it in the next milestone.
